tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -149,6 +149,24 @@ Feature: full text search
       | spaces           |
 
 
+  Scenario Outline: search version restored files using a tag
```

Comment:
I don't think `version` is needed here
```suggestion
  Scenario Outline: search restored files using a tag
```

Reply:
there is the scenario with the same title L133

Reply:
Then maybe 
```suggestion
  Scenario Outline: search restored version of a file using a tag
```